### PR TITLE
rubocop fixes and install method

### PR DESCRIPTION
**Pull request by TD-4242**
This includes a lot of fixes for rubocop style complaints.

Also added an install_method attribute that can be set to remote file and the cookbook will attempt to download from a provided url rather than having to add the file to the cookbook.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,40 +24,46 @@ Attributes
...
+    <td><tt>[]</tt></td>
   </tr>
 
   <tr>
-    <td><tt>['storm'][:nimbus_ip]</tt></td>
+    <td><tt>['storm']['nimbus_ip']</tt></td>
     <td>String</td>
-    <td>Storm nimbus server IP address</td>
-    <td><tt>192.168.50.4</tt></td>
+    <td>Storm nimbus server IP address, or insure nimbus dns name resolves to your nimbus server</td>
```
Comment (Lewuathe) [inline README.md:57]: Is this typo?

```
s/insure/ensure
```

Comment (TD-4242) [inline README.md:57]: I think Geico has a policy for this, or yes it's a typo.

**Comment by Lewuathe** [comment]
@TD-4242 LGTM :+1:  Thank you so much!

**Comment by TD-4242** [comment]
I started using this template method for a json config file as well.  I may try updating the dse one for cassandra.yaml too.

**Pull request merged**